app/services/users.js

```diff
@@ -12,3 +12,14 @@ exports.createUser = async payload => {
     throw errors.databaseError('Something went wrong trying to save into the DB');
   }
 };
+
+exports.getUsers = async pagination => {
+  try {
+    const allUsers = await User.findAll({ offset: 0, limit: pagination });
```

Comment:
You should be able to control both the offset and the limit from the query parameters. I recommend using `page` and `limit` as the query params instead of `pagination` and find a way to calculate the offset based on the `page` and `limit`